test/Datasource.test.js

```diff
@@ -57,10 +62,55 @@ describe('Test all datasouses from ../src/lib/datasouce', () => {
         });
     
         it('should resolve with null', ()=> {
-          return dataSource.delete().then((res) => {
-            expect(res).toEqual(null);  
+          return dataSource.delete('key').then((res) => {
+            expect(res).toEqual(true);  
           });
         });
       })
 
+      describe('#chaining', () => {
```

Comment:
chaining is not a method, so do not start it with a hash

Reply:
ok